app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @posts = User.find_by(id: params['user_id'])
```

Comment:
- [ ] Just as the previous reviewer noted, kindly rename your `@posts` to `@user` variable because it holds the user with id `params['user_id'] and it's not an actual post 👍